### PR TITLE
Rahwa-Ghebremichael/London/Class-8/JS-Core-2/ Week-1

### PR DESCRIPTION
**Pull request by rahwafesseha**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

File: 1-exercises/A-accessing-values/exercise3.js [hunk 1]
```diff
@@ -21,9 +21,9 @@ let basketballTeam = {
 */
 
 // write code here
- let topPlayersBasketball = basketballTeam["topPlayers"];
-console.log(topPlayersBasketball);
-/* EXPECTED RESULT
+ basketballTeam.topPlayers.sort().forEach((player) => console.log(player));
+
```
Comment (Monika39-maker) [inline 1-exercises/A-accessing-values/exercise3.js:25]: good use of forEach.

File: 2-mandatory/1-recipes.js [hunk 2]
```diff
@@ -23,37 +23,67 @@
...
+  servings: 1,
+  ingredients: ["chicken", "single cream", "garlic", "garam masala", "ginger", "butter"],
+};
+
+console.log(fifthRecipe.title);
+console.log(`Serves: ${fifthRecipe.servings}`);
+console.log(`Ingredients:`);
+fifthRecipe.ingredients.forEach((ingredient) => {
+  console.log(ingredient);
+});
```
Comment (Monika39-maker) [inline 2-mandatory/1-recipes.js:89]: well done!!

**Review by Monika39-maker — APPROVED** [review]

File: 1-exercises/A-accessing-values/exercise3.js [hunk 3]
```diff
@@ -21,12 +21,12 @@ let basketballTeam = {
 */
 
 // write code here
+ basketballTeam.topPlayers.sort().forEach((player) => console.log(player));
```
Comment (mahsa2) [inline 1-exercises/A-accessing-values/exercise3.js:24]: Nice chaining 👍

File: 1-exercises/C-undefined-properties/exercise.js [hunk 4]
```diff
@@ -36,3 +36,4 @@ let myPet = {
 };
 
 console.log(myPet.getName());
+//The method getName does not console.log anything.
```
Comment (mahsa2) [inline 1-exercises/C-undefined-properties/exercise.js:39]: We could also say that it doesn't `return` anything on line 34

File: 1-exercises/D-object-methods/exercise.js [hunk 5]
```diff
@@ -9,7 +9,11 @@
 
 let student = {
   // write code here
-}
+   getName: function (string) {
+     console.log(`Student name: ${string}`);
```
Comment (mahsa2) [inline 1-exercises/D-object-methods/exercise.js:13]: Nice use of string interpolation 💯

**Review by mahsa2 — COMMENTED** [review]
Well done Rahwa 👍 nice to read your code and see how much you've learned so far 🥇

**Pull request closed without merging**